src/test/java/com/exactpro/th2/common/event/bean/TestVerification.java

```diff
@@ -53,7 +54,8 @@ public void testSerializationSimpleVerification() throws IOException {
                 "        \"status\": \"PASSED\",\n" +
                 "        \"key\": false,\n" +
                 "        \"actual\": \"value\",\n" +
-                "        \"expected\": \"*\"\n" +
+                "        \"expected\": \"*\",\n" +
+                "        \"hint\": \"Hint for not empty field\"\n" +
```

Comment:
I suggest you to leave one verification entry without hint because it is not required